test-utils/runtime/Cargo.toml

```diff
@@ -76,7 +76,6 @@ std = [
 	"log/std",
 	"sp-offchain/std",
 	"sp-core/std",
-	"sp-core/std",
```

Comment:
@ggwpez does your tool detect duplicates like this?

Reply:
Ah, does not do that yet. But the `featalign` from Xavier should.

Reply:
what's `featalign`?

Reply:
Similar tool https://github.com/hack-ink/cargo-featalign :wink: